### PR TITLE
fix: remove checkov_skips for logdna agent

### PR DESCRIPTION
**Pull request by maheshwarishikha**
### Description

Remove checkov_skips for logdna agent

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: chart/logdna-agent/values.yaml [hunk 1]
```diff
@@ -12,24 +12,24 @@ secret:
...
-  - checkov.io/skip15: CKV_K8S_37 # "Minimize the admission of containers with capabilities assigned" - ignoring as chart not owned by us
-  - checkov.io/skip16: CKV_K8S_31 # "Ensure that the seccomp profile is set to docker/default or runtime/default" - ignoring as chart not owned by us
-  - checkov.io/skip17: CKV_K8S_15 # "Image Pull Policy should be Always" - when imagePullPolicy is Always, the image is pulled every time the pod is started. While this can help ensure deployment consistency, it can also prevent a pod from starting if the image registry can not be reached.
-  - checkov.io/skip18: CKV_K8S_13 # "Memory limits should be set" - ignoring as chart not owned by us
-  - checkov.io/skip19: CKV_K8S_10 # "CPU requests should be set" - ignoring as chart not owned by us
-  - checkov.io/skip20: CKV_K8S_16 # "Container should not be privileged" - ignoring as chart not owned by us
-  - checkov.io/skip21: CKV_K8S_28 # "Minimize the admission of containers with the NET_RAW capability" - ignoring as chart not owned by us
+# checkov_skips:
```
Comment (ocofaigh) [inline chart/logdna-agent/values.yaml:15]: since it passes, remove the code comments and re-trigger the pipeline

Comment (maheshwarishikha) [reply chart/logdna-agent/values.yaml]: done

**Review by jojustin — APPROVED** [review]

**Pull request merged**